src/gt4py/next/program_processors/runners/roundtrip.py

```diff
@@ -209,7 +209,7 @@ def execute_roundtrip(
     *args: Any,
     column_axis: Optional[common.Dimension] = None,
     offset_provider: dict[str, embedded.NeighborTableOffsetProvider],
-    debug: bool = False,
+    debug: bool = config.DEBUG,
```

Comment:
Note that this means changing the `config.DEBUG` flag after this module is imported has no effect. Maybe retrieving the value in the function is what we actually want.